packages-exp/auth-compat-exp/index.rn.ts

```diff
@@ -23,5 +23,9 @@
  */
 
 import { testFxn } from './src';
+import { AsyncStorage } from 'react-native';
+import { ReactNativePersistence } from '@firebase/auth-exp/src/core/persistence/react';
```

Comment:
Any thoughts on renaming the files and modules involved to `react-native` instead of `react`? If no users will ever see it, maybe `rn`? Don't want to confuse anything for React users. Even if it's not user-facing, it could be a bit confusing to a contributor or someone trying to help debug by looking at our source code.

Reply:
great idea!